### PR TITLE
Fix hdfs write

### PR DESCRIPTION
**Pull request by 244372610**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by yxxhero — APPROVED** [review]
LGTM

File: pkg/source/hdfsprotocol/hdfs_source_client.go [hunk 1]
```diff
@@ -61,14 +61,14 @@ type hdfsSourceClient struct {
 
 // hdfsFileReaderClose is a combination object of the  io.LimitedReader and io.Closer
 type hdfsFileReaderClose struct {
-	limited io.Reader
-	c       io.Closer
+	reader io.Reader
```
Comment (hyy0322) [inline pkg/source/hdfsprotocol/hdfs_source_client.go:64]: I don't think rename limited to reader is a good idea

Comment (jim3ma) [reply pkg/source/hdfsprotocol/hdfs_source_client.go:64]: How about `limited -> limitedReader` ?

**Review by jim3ma — APPROVED** [review]
LGTM

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**